lib/matplotlib/testing/compare.py

```diff
@@ -355,8 +355,8 @@ def crop_to_same(actual_path, actual_image, expected_path, expected_image):
     # clip the images to the same size -- this is useful only when
     # comparing eps to pdf
     if actual_path[-7:-4] == 'eps' and expected_path[-7:-4] == 'pdf':
-        aw, ah = actual_image.shape
-        ew, eh = expected_image.shape
+        aw, ah, ad = actual_image.shape
```

Comment:
According to coverage it was not run before https://codecov.io/gh/matplotlib/matplotlib/compare/f5f3d793a7a14cfd186f76a987826e58c3337437...51f070536cdb8b89552f03091b3068a6fc45e5a0/src/lib/matplotlib/testing/compare.py#L357